### PR TITLE
Adding AWS credential types

### PR DESCRIPTION
**Pull request by belialboy**
Adding AWS Access Key and AWS Secret Access Key to the supported regex patterns

### CODE REVIEW

**Comment by bee-san** [comment]
Thank you for this! This needs tests, feel free to add them or we can do it later :)

**Comment by belialboy** [comment]
My bad. I've added the tests for you.

File: pywhat/Data/regex.json [hunk 1]
```diff
@@ -151,6 +151,30 @@
...
     },
+    {
+      "Name": "Amazon Web Services Access Key",
+      "Regex": "(?<![A-Z0-9])[A-Z0-9]{20}(?![A-Z0-9])",
+      "plural_name": false,
+      "Description": "Access key used to authenticate with the AWS API",
+      "Rarity": 1,
+      "URL": null,
+      "Tags": [
+         "Credentials",
```
Comment (bee-san) [inline pywhat/Data/regex.json:162]: I just remembered!! We have an `AWS` tag, mind adding that? And if you spot any other AWS regex without it please add it :) AWS has so many (albeit, confusing) IDs that it needs its own tag haha!

Comment (bee-san) [reply pywhat/Data/regex.json:162]: other than this, looks good to go 👍🏻

Comment (belialboy) [reply pywhat/Data/regex.json:162]: Can do. Would you like me to add some of the other AWS types like security group, EC2 instance, organization ID? They'll be less common, but do have a fixed format.

Comment (bee-san) [reply pywhat/Data/regex.json:162]: Sure, if you don't mind! :)

Comment (belialboy) [reply pywhat/Data/regex.json:164]: Done... I'll see if I can think of others on my travels and will update as I go. Ok?

**Comment by piatrashkakanstantinass** [comment]
Merge conflicts😢

**Comment by bee-san** [comment]
If possible, please fix the merge conflicts && remove the descriptions -- we try to only include them if it's needed (I.E. the name is not descriptive of what it is)

**Comment by amadejpapez** [comment]
Seems like some newly added API key formats are first getting matched as AWS ones. You can change tests to this format:
`assert "Amazon Web Services Access Key" in res` as currently test is checking only the first match.

**Comment by amadejpapez** [comment]
I have fixed tests and removed `AWS Access Key ID` as I think it is the same as newly added `Amazon Web Services Access Key`.

**Pull request merged**